.env.example

```diff
@@ -1,3 +1,4 @@
 REACT_APP_API_PREFIX="api"
 SENTRY_DSN=""
 REACT_APP_GLIFIC_API_PORT=4000
+REACT_APP_SIMULATOR_CONTACT_NUMBER="9876543210"
```

Comment:
Why do we need to define the phone number here? Since it will be used internally should we define it in constant?

Reply:
this is something we're unsure of what exactly would be the number. So added this flexibility that it can easily changed in Glific installations by others

Reply:
We are not sending actual messages, so in a way, the number does not matter right?

Reply:
the number needs to be tracked by the backend as it is treated as a contact and the messages are stored. I think we can keep it fixed in config.

Reply:
Can you remove "REACT_APP_SIMULATOR_CONTACT_NUMBER" from .env.example as it is defined in constant